packages/agreement-process/src/resources/templates/documents/agreementContractTemplate.html

```diff
@@ -1,97 +1,137 @@
+    <!-- Intestazione di pagina -->
+    <div class="header">
+      <div class="logo">
+        <img src="../assets/media/logo_pagopa.png" />
+      </div>
     </div>
-  </div>
 
-  <!-- Titolo del documento -->
-  <h1>Richiesta di fruizione</h1>
+    <!-- Titolo del documento -->
+    <h1>Richiesta di fruizione</h1>
 
-  <!-- Intro legale -->
-  <div class="legal-intro">
-    In data <strong>{{todayDate}}</strong> alle ore
-    <strong>{{todayTime}}</strong>, l’Infrastruttura ha ricevuto la richiesta
-    di fruizione contraddistinta da id <strong>{{agreementId}}</strong> (di
-    seguito “Richiesta di Fruizione”), inviata dall’aderente
-    <strong>{{consumerText}}</strong> (di seguito “Fruitore”), tramite
-    l’operatore amministrativo <strong>{{submitter}}</strong>, per l’E-service
-    <strong>{{eServiceName}}</strong> reso disponibile dall’aderente
-    <strong>{{producerText}}</strong>
-    (di seguito “Erogatore”) di cui il Fruitore soddisfa i seguenti requisiti
-    di fruizione per lo stesso previsti dall’Erogatore.
-  </div>
-  <div class="content">
-    <h2>Attributi dichiarati</h2>
+    <!-- Intro legale -->
+    <div class="legal-intro">
+      In data <strong>{{todayDate}}</strong> alle ore
+      <strong>{{todayTime}}</strong>, l’Infrastruttura ha ricevuto la richiesta
+      di fruizione contraddistinta da id <strong>{{agreementId}}</strong> (di
+      seguito “Richiesta di Fruizione”), inviata dall’aderente
+      <strong>{{consumerText}}</strong> (di seguito “Fruitore”), tramite
+      l’operatore amministrativo <strong>{{submitter}}</strong>, per l’E-service
+      <strong>{{eServiceName}}</strong> reso disponibile dall’aderente
+      <strong>{{producerText}}</strong>
+      (di seguito “Erogatore”) di cui il Fruitore soddisfa i seguenti requisiti
+      di fruizione per lo stesso previsti dall’Erogatore.
+    </div>
+    <div class="content">
+      {{#if declaredAttributes}}
 
-    {{{declaredAttributes}}}
+      <h2>Attributi dichiarati</h2>
+      {{#each declaredAttributes}}
+      <div>
+        In data
+        <strong>{{this.assignmentDate}}</strong>
+        alle ore
+        <strong>{{this.assignmentTime}}</strong>, l’infrastruttura ha registrato
+        la dichiarazione del fruitore di possedere l’attributo dichiarato
+        <strong>{{this.attributeName}}</strong>, contraddistinto da id
+        <strong>{{this.attributeId}}</strong>, necessario a soddisfare il
+        requisito di fruizione stabilito dall’erogatore per l’accesso
+        all’e-service.
+      </div>
+      {{/each}} {{/if}} {{#if verifiedAttributes}}
 
-    <h2>Attributi verificati</h2>
+      <h2>Attributi verificati</h2>
 
-    {{{verifiedAttributes}}}
+      {{#each verifiedAttributes}}
+      <div>
+        In data
+        <strong>{{this.assignmentDate}}</strong>
+        alle ore
+        <strong>{{this.assignmentTime}}</strong>, l’infrastruttura ha registrato
+        la verifica del delegato all’erogazione che il fruitore possegga
```

Comment:
This part change in the standard agreement

```suggestion
        la verifica {{#if delegate}} del delegato {{else}} dell’erogatore {{/if}} all’erogazione che il fruitore possegga
```